### DISCUSSION TITLE
You should write somewhere in the documentation that the user should not load Tailwindcss Form plugin

### DISCUSSION THREAD

**Discussion by joffreypersia** [livewire/flux#269] [Feature Requests] [answered]
Hi, 

I had in my tailwind.config.js the plugin tailwindcss form, and it doesn't work well with Flux. I found a side effect with an input group where the input group prefix and input text had rounded edges between each other. 
```
<flux:input.group>
        <flux:input.group.prefix>https://</flux:input.group.prefix>

        <flux:input wire:model="website" placeholder="example.com" />
</flux:input.group>
```

**Comment by jpangborn** [comment]
Thanks for this. I was trying to figure out why the focus states on `flux:input` didn't match what the fluxui.dev site showed. The Tailwind Forms plugin was the culprit. Removing it fixed the focus state styles.

**Comment by amitsamtani** [comment]
Seems like they added this to the [Flux installation docs](https://fluxui.dev/docs/installation): 

> Note: Flux is not compatible with the tailwindcss/forms plugin.

Does this mean, we can not use Flux with [FilamentPHP Forms](https://filamentphp.com/docs/3.x/forms/installation) as it makes use of that plugin: 

```bash
npm install tailwindcss @tailwindcss/forms @tailwindcss/typography postcss postcss-nesting autoprefixer --save-dev
```

I have been working with Flux and FilamentPHP Tables and Forms and things were looking good until I realized my Filament badge was not rendering correctly. Is something in the works to make it compatible?

**Reply by ju5t** [reply]
Essentially you have competing UI frameworks. `@tailwindcss/forms` is an opinionated package, as is Flux. And Filament adds its UI flavor on top of that. Mixing everything together is a recipe for disaster.

Personally, I wouldn't expect this to see this resolved in the near future.

**Reply by amitsamtani** [reply]
Thanks for the response @ju5t. Makes sense.

**Comment by joshhanley** [comment, marked as answer]
Closing to close this as we added compatibility with Tailwind Forms in Flux v1.0.30
